cpp/include/raft/sparse/selection/knn_graph.hpp

```diff
@@ -16,8 +16,8 @@
 
 #pragma once
 
-#include <raft/linalg/distance_type.h>
 #include <cstdint>
+#include <raft/linalg/distance_type.h>
 #include <raft/sparse/coo.hpp>
 #include <raft/sparse/selection/detail/knn_graph.cuh>
 
```

Comment:
```suggestion
#include <raft/linalg/distance_type.h>
#include <raft/sparse/coo.hpp>
#include <raft/sparse/selection/detail/knn_graph.cuh>

#include <cstdint>
```
Should start to follow clang-tidy style of includes from "closest" to "furthest" in different blocks (see https://llvm.org/docs/CodingStandards.html#include-style)